api/v1alpha1/storageconsumer_types.go

```diff
@@ -58,6 +58,13 @@ type CephResourcesSpec struct {
 	CephClients map[string]string `json:"cephClients,omitempty"`
 }
 
+type ConsumerClusterStatus struct {
+	// Timestamp of last heartbeat received from consumer
+	LastHeartbeat metav1.Time `json:"lastHeartbeat,omitempty"`
+	// Phase of corresponding consumer storageCluseter
+	ExternalStorageClusterPhase string `json:"externalStorageClusterPhase,omitempty"`
```

Comment:
a consumer is an external client to the provider? just call this as StorageClusterStatus.

Reply:
Updated in latest commit